index.html

```diff
@@ -126,6 +126,69 @@ <h2 class="text-center">Babel transforms your JavaScript</h2>
     </div>
   </div>
 
+    <hr class="featurette-divider">
+
+    <div class="row featurette">
+      <div class="col-md-6">
+        <h2>Polyfill</h2>
+        <p>Since Babel only transforms syntax (like arrow functions), you can use babel-polyfill in order to support new globals such as Promise or new native methods like String.padStart (left-pad). It uses <a href="https://github.com/zloirock/core-js">core-js</a> and <a href="http://facebook.github.io/regenerator/">regenerator</a>. Check out our <a href="/docs/usage/polyfill">babel-polyfill</a> docs for more info.</p>
+
+        <p>You can install the polyfill with</p>
+  {% highlight javascript %}
+  $ npm install --save-dev babel-polyfill
+  {% endhighlight %}
+        </p>Use it by requiring it at the top of the entry point to your application.</p>
```

Comment:
I think I would update http://babeljs.io/docs/usage/polyfill/ to give a webpack example like what you were saying